src/main/java/binary_search/OccuranceOfElement.java

```diff
@@ -0,0 +1,37 @@
+package binary_search;
+    public class OccuranceOfElement {
+        public int countOfElement(int[] nums,int target){
+            int start=0;
+            int end=nums.length-1;
```

Comment:
corner case handling is missing. What if the count is 0 or 1. You do not want to execute any logical calculation is such cases to make your code optimised.